src/controllers/index.js

```diff
@@ -1,24 +1,39 @@
-const express = require('express');
-const queries = require('../queries');
+const express = require("express");
+const queries = require("../queries");
+const bodyParser = require("body-parser");
 const router = express.Router();
-const path = require('path');
+const path = require("path");
+// var sqlinjection = require("sql-injection");
```

Comment:
Please can you delete commented out code if it's not being used in Production

Reply:
cleaned the unnecessary comments